specification/ai/Azure.AI.Projects/agents/routes.tsp

```diff
@@ -51,20 +51,20 @@ op listAgents is Azure.Core.Foundations.Operation<
 /**
  * Retrieves an existing agent.
  *
- * @param assistantId The ID of the agent to retrieve.
+ * @param agentId The ID of the agent to retrieve.
  * @returns The requested agent instance.
  */
 #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "not yet an Azure operation"
 #suppress "@azure-tools/typespec-azure-core/no-operation-id" "non-standard operations"
 #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "not yet versioned"
 #suppress "@azure-tools/typespec-azure-core/use-standard-names" "mirrored API name parity"
 @get
-@route("/assistants/{assistantId}")
+@route("/assistants/{agentId}")
```

Comment:
Should route match too ?

Reply:
@singankit are you asking about the "assistants" word in the route "/assistants/{...}"? That should stay as is. There is no change in the wire protocol... we still use "assistants" in the route and "assistant_id" in request/response JSON payloads. This PR change just affects the public APIs of emitted SDKs.